build.gradle.kts

```diff
@@ -63,6 +63,8 @@ tasks.register("printSignedPluginZipPath") {
   }
 }
 
+val configCheckerDirectory = rootProject.file("config/checker")
```

Comment:
Nit: pls wrap into an `extra(...)` as stuff above is, and refer from CheckerFramework.kt in `-Astubs=` flag